projects/forms/src/components/calendar/components/calendar-cell-empty/calendar-cell-empty.component.scss

```
@@ -1,12 +1,12 @@
-@use 'functions' as *;
+@use 'functions';
 @use '../../styles/calendar.styles' as *;
```

```suggestion
@use '../../styles/calendar.styles';
```
попробуй убрать и использовать ниже как calendar.base-cell()